tests/provision/container/basic/main.fmf

```diff
@@ -0,0 +1,7 @@
+summary: Provision a fedora container and run a simple command
+description:
+    Make sure that the simple use case of starting a container,
+    running a simple command in it and stopping it works fine and
+    does not trigger any AVC denials.
+tag+:
```

Comment:
Would adding `provision-only` and `provision-container` make sense?

Reply:
Yeah, makes sense to have them for all three `provision/container` tests, fixed in d74f7b8c.